.vitepress/theme/keycap-palette.css

```
@@ -0,0 +1,223 @@
+ *   Light BG:     #f8f9fa (warm paper)
+ *   Slate:        #353a40 (secondary surfaces)
+ *
+ * Contrast: All text/bg combos meet WCAG AA (4.5:1+)
+ * ═══════════════════════════════════════════════════════════════ */
+
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap');
+
+/* ── Shared Tokens ── */
+
+:root {
+  --vp-font-family-base: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.4.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)